### DISCUSSION TITLE
navigation shortcut collides with macOS default hotkey (in builtin markdown editor)

### DISCUSSION THREAD

**Discussion by cmyr** [community/community#6049] [Projects and Issues] [answered]
As of recently,  there has been a keyboard shortcut added to insert backticks around the currently selected word. According to the tooltip, this shortcut is <kbd>cmd</kbd><kbd>e</kbd>. This is okay, but for me (currently on macos 10.14.6, in both a recent safari and the latest chrome) this is also accepting <kbd>ctrl</kbd><kbd>e</kbd>.

ctrl+e is a standard keybinding in the [Cocoa Text System](https://www.hcs.harvard.edu/~jrus/site/system-bindings.html). On any well-behaved text-input field on macOS, ctrl+e should move the cursor to the end of the current line.

In any case, my sense is that the bug here is that <kbd>ctrl</kbd><kbd>e</kbd> is being handled at all; on mac you should be fine just handling <kbd>cmd</kbd><kbd>e</kbd>, although this is also a standard keybinding (Edit -> use selection for find).

![Screen Shot 2021-10-06 at 2 32 04 PM](https://user-images.githubusercontent.com/3330916/136262423-9783c6e3-5402-401a-8ad3-84ddd4b8c27a.png)
*tooltip*

![Screen Shot 2021-10-06 at 2 33 15 PM](https://user-images.githubusercontent.com/3330916/136262567-4b4bc37f-86e2-4597-b205-734b2498ae12.png)
*Edit->Find*

**Comment by lukaszkorecki** [comment]
This also breaks macOS conventions inherited from Emacs/readline (e.g. `Ctrl-E` to jump to the end of the line)

**Comment by cmyr** [comment, marked as answer]
Fixed per https://github.com/github/feedback/discussions/6048, thank you!

**Reply by moble** [reply]
This does not appear to be fixed on Firefox...
